### PR TITLE
Many updates to UI strings

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
Remove [possibly racist terms](http://garysaid.com/are-the-terms-whitelist-and-blacklist-racist/), simplify language, enhance clarity, improve consistency, and fix errors.

### CODE REVIEW

**Comment by ukanth** [comment]
@Gitoffthelawn , Can you fix the issue and send pull request again ? 

/home/travis/build/ukanth/afwall/aFWall/src/main/res/values/strings.xml:6:6: Error: The processing instruction target matching "[xX][mM][lL]" is not allowed.

You just need to remove the top comment.

**Comment by Gitoffthelawn** [comment]
Thanks.  I'll do that right away.

**Comment by Gitoffthelawn** [comment]
Before you merge, can you tell me if the extra intra-line spaces in lines 83 and 84 can be deleted?

If so, just let me know, and I'll clean that up.

File: aFWall/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -99,271 +104,267 @@
...
+    <!-- Advanced Settings -->
     
-        init.d
-    -->
-    <string name="success_initd">Successfully installed startup script</string>
+    <!-- init.d -->
+    <string name="success_initd">Startup script installed</string>
     <string name="unable_initd">Unable to install AFWall+ startup script. Please verify that your ROM/Kernel supports init.d/su.d</string>
-    <string name="remove_initd">Successfully removed startup script</string>
+    <string name="remove_initd">Sartup script removed</string>
```
Comment (ukanth) [inline aFWall/src/main/res/values/strings.xml]: @Gitoffthelawn There is a spell mistake. Kindly correct it.

**Comment by CHEF-KOCH** [comment]
Firewall Blacklist (or blacklists) aren't racist. The context is important and this entire project or the app itself not even in any way related to this.

besides this useless change the rest seems okay to merge.

**Comment by Gitoffthelawn** [comment]
Good feedback.  Thanks!

Regarding _iptables_: The most common use is lowercase.  Wikipedia even uses lowercase for it when it appears at the beginning of a sentence.  I agree completely that _ip_ is normally capitalized, but in this context it is not.  The actual file is lowercase in all the case-sensitive operating systems in which it is used.

Regarding using lowercase for the first letter of words beyond the first word of UI strings: Google is inconsistent on this, but the use of capitalizing the first word and using lowercase for the rest of the words seems more prevalent in the current app ecosystem.  For example, take a look at all the menus and items in the stock Android _Settings_ app.

I'm looking through all my Android UI/UX design references, and I'm not seeing any "official" recommendation... if you happen to find one that says one way or another, that would be fantastic and much appreciated!

File: aFWall/src/main/res/values/strings.xml (hunk 1 above)
Comment (Gitoffthelawn) [inline aFWall/src/main/res/values/strings.xml]: @ukanth Done.  Thank you!

**Comment by Gitoffthelawn** [comment]
@ukanth The summary string for _Manage Profiles_, and the reference to it, were missing from all the string files.

I added it to the primary string file and added the reference to it in the menu generation code.

What is the best way to add it to crowdin to get translations to update all the translation string files?

**Comment by CHEF-KOCH** [comment]
> What is the best way to add it to crowdin to get translations to update all the translation string files?

Nothing, because you already added it here so he can directly merge it. 

> Regarding iptables

iptables is executed in lowercase, that's why it's wrongly called 'iptables'. I think overall it not matters much because it's the same just in uppercase. Make what your want

**Pull request merged**